package.json

```diff
@@ -197,7 +196,7 @@
     "semver": "7.5.3",
     "shelljs": "0.8.5",
     "sinon": "7.3.2",
-    "snap-shot-it": "7.9.3",
+    "snap-shot-it": "7.9.10",
```

Comment:
I was having issues with snapshot and though this would fix it. Seems like a good change even though it's not needed for this update.